### DISCUSSION TITLE
Vehicules driving seats considered as respawn point after death

### DISCUSSION THREAD

**Discussion by Pochtron** [Vdauphin/HeartsAndMinds#1233] [Question and answer]
![image](https://user-images.githubusercontent.com/76669953/140630197-8cdcafeb-5c40-4fc0-9dfd-42e656165707.png)

Hello VDauphin, Thanks to your help, almost everything is working on one of my H&M campaign but I still have trouble with respawns. Indeed, when I start the game, I spawn normaly at marker "east respawn" into my base but when I die, it giving me the choice, no matter what kind of respawn I choose in Attribute/multiplayer options in Eden Editor the ability to spawn at the "east respawn" but also at almost all the vehicules btc_veh.... type driving seat that I spread on the map for scenario purpose. Where do you think it could come from? I'm currently with opfor nation as player and US army woodland from RHS as ennemy.

Thanks a lot in advance for your advice!

Pochtron

**Comment by Vdauphin** [comment]
Hello,
https://github.com/Vdauphin/HeartsAndMinds/blob/46c2853b3b090447014bd05f87468ddd0a1efd6c/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L50-L54
Cheers
